content/en/blog/2021/wasm-progress/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Progress of Envoy and Istio WebAssembly Extensibility
+description: An update on Envoy and Istio WebAssembly based extensibility effort.
+publishdate: 2021-02-09
+attribution: "Pengyuan Bian (Google)"
+keywords: [wasm,extensibility,WebAssembly]
+---
+
+Back to Istio 1.5 release, Istio introduced [WebAssembly based extensibility](/blog/2020/wasm-announce/) support.
+Over the past year, Istio, Envoy, and Proxy-Wasm communities have continued the effort to make WebAssembly extensibility stable, reliable, and easy to adopt.
+In this blog post, we will walk through the updates in this area at Istio 1.9 release.
+
+## Proxy-Wasm (WebAssembly) support merged in upstream Envoy
+
+The initial development work of Envoy WebAssembly runtime happened at `envoy-wasm` fork.
+Over time, we collected great feedback from community early adopters, as well as Istio first class Wasm extensions development, which helped us to continuously improve the runtime.
+Since Istio 1.8, WebAssembly support was merged into the Envoy main tree. This is a significant milestone for Istio and Envoy Wasm efforts, since it indicates that:
+
+* The runtime is ready for wider adoption.
+* The programming ABI/API, extension configuration API, as well as runtime behavior are becoming stable.
+* Better community support moving forward.
+
+## Istio Wasm Extension Ecosystem
+
+As one of the early adopters of Envoy WebAssembly runtime, Istio extensibility working group gained a lot of experience via developing several first class extensions, such as metadata exchange, Prometheus stats, and attribute generation.
```

Comment:
Should we link to the WG page on community repo?  And should we link to any docs pages on the extensions (for audiences that might not know what metadata exchange is, for instance) ?

Reply:
Where is the working group page? I cannot find it in community repo.. and closest I can find is https://github.com/istio/community/blob/master/WORKING-GROUPS.md